tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/ProcessBasicCheck.cs

```diff
@@ -540,13 +579,13 @@ private static bool TracingWithBundle(string?[] profilerPathValues, int processI
             const string uninstallKey64Bit = @"SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\";
             const string uninstallKey32Bit = @"SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall";
 
-            if (GetLocalMachineSubKeyVersion(uninstallKey64Bit, datadog64BitProgram, out var tracerVersion))
+            if (GetLocalMachineSubKeyVersion(uninstallKey64Bit, datadog64BitProgram, out var tracerVersion, registryService))
```

Comment:
Did this method change? I'm confused why we're adding a new parameter to the method in this PR

Reply:
This looks like it was a bug - there's a default parameter in `GetLocalMachineSubKeyVersion` that was not getting passed through. I think it's fine to fix that here (maybe also make the parameters non-default!)

Reply:
Yeah forgot to mention it on the commit but I noticed this bug!